### PR TITLE
Support for React ^15.6

### PR DESCRIPTION
**Pull request by tysonbattistella**

### CODE REVIEW

File: src/index.spec.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+import CoinbaseCommerceButton from './index';
 
 test('it renders and matches its snapshot', () => {
-  const wrapper  = mount(
+  const wrapper  = shallow(
     <CoinbaseCommerceButton checkoutId={'aaaa'}/>
   );
-  expect(wrapper.children().find('iframe').length).toEqual(0);
-  expect(wrapper.children().find('button').length).toEqual(1);
+  expect(wrapper.children().first().children().find('IFrame').length).toEqual(0);
```
Comment (guacamoli) [inline src/index.spec.js]: Not 100% sure but I think you can pass children() a selector.

File: src/components/IFrame.js [hunk 2]
```diff
@@ -117,8 +109,8 @@ export default class IFrame extends React.Component<Props, State> {
           <div className="commerce-loading-spinner"/>
         ) : (null)}
         <iframe
+          onLoad={this.handleIFrameLoaded}
```
Comment (guacamoli) [inline src/components/IFrame.js]: Did you confirm that this works?

File: src/components/IFrame.js [hunk 3]
```diff
@@ -29,10 +29,6 @@ export default class IFrame extends React.Component<Props, State> {
     super(props);
 
     this.origin = 'https://commerce.coinbase.com';
-
-    this.isValidMessage = this.isValidMessage.bind(this);
```
Comment (guacamoli) [inline src/components/IFrame.js]: semicolons on line 16 and 20

File: src/index.spec.js (hunk 1 above)
Comment (tysonbattistella) [reply src/index.spec.js]: Unfortunately not :(

**Pull request merged**